app/plugin_api_test.go

```diff
@@ -731,6 +745,93 @@ func TestPluginAPIInstallPlugin(t *testing.T) {
 	assert.True(t, found)
 }
 
+func TestInstallPlugin(t *testing.T) {
+	th := Setup(t).InitBasic()
+	defer th.TearDown()
+
+	path, _ := fileutils.FindDir("tests")
+	ts := httptest.NewServer(http.FileServer(http.Dir(path)))
+	defer ts.Close()
+	downloadURL := ts.URL + "/testplugin.tar.gz"
+
+	var pluginJson map[string]interface{}
+	err := json.Unmarshal([]byte(fmt.Sprintf(`{"DownloadURL": "%s"}`, downloadURL)), &pluginJson)
```

Comment:
Why `json.Unmarshal` instead of just populating `pluginJson` directly?

Reply:
Oops, totally my bad! I remember that I've just copied this code from the existing tests somewhere. Will be sending a fix commit in a bit.